Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -23,6 +23,8 @@ class SFConnectionIT : SFBaseTest
     {
         private static SFLogger logger = SFLoggerFactory.GetLogger<SFConnectionIT>();
 
+        private static readonly string ConnectionFailureErrorCode = "08006";

```

Comment:
It is possible to use SnowflakeDbException.CONNECTION_FAILURE_SSTATE static variable? Im not sure because it is internal static string.

Reply:
It should work - changed